boards/arm/nucleo_f446re/nucleo_f446re.dts

```diff
@@ -56,7 +57,7 @@
 
 &i2c1 {
 	pinctrl-0 = <&i2c1_scl_pb8 &i2c1_sda_pb9>;
-	status = "okay";
+	status = "disabled";
```

Comment:
Not needed, and "harmful".
Both dts nodes can live enabled together as valid pre-configuration, but ... the point is that CONFIG_I2C should not be enabled with CONFIG_CAN.
By disabling this node, people using arduino i2c through shields for instance will face issues of missing i2c node.
Then, if possible, to avoid conflict, I'd suggest to move the can device to other pins (as i2c is more commonly used)